rts/lra/lra-client/src/main/java/io/narayana/lra/client/Current.java

```diff
@@ -156,7 +156,10 @@ public static void updateLRAContext(ClientRequestContext context) {
         if (lraId != null) {
             context.getHeaders().putSingle(LRA_HTTP_HEADER, lraId);
         } else {
-            Object lraContext = context.getProperty(LRA_HTTP_HEADER);
+            Object lraContext = getState("suspendedLRA");
```

Comment:
Is it not worth using a "final static String" here?

Reply:
+1

Reply:
I am going to do the same for the resource paths as well which are currently defined as literal strings but are referenced from different places (in the resource @PATH annotations and in the SpecIT tests).